app/components/display-package-code.js

```diff
@@ -0,0 +1,5 @@
+import Ember from "ember";
+
+export default Ember.Component.extend({
+  classNames: ["display-package-code"]
```

Comment:
is this required?

Reply:
Yes, otherwise you have to specifically pass class everytime you call the component